testsuite/tests/libratest/smoke_test.rs

```diff
@@ -146,15 +121,20 @@ impl TestEnvironment {
         .unwrap()
     }
 
-    fn get_validator_ac_client(
+    fn get_client(
```

Comment:
As someone who just learned about the smoke tests, I appreciate getting rid of a lot of the extra words!